app/components/certificate/certificate-request.tsx

```diff
@@ -16,24 +16,38 @@ export default function CertificateRequestView({
   onRequest,
 }: CertificateRequestViewProps) {
   return (
-    <Flex flexDirection="column" gap="5" width="4xl">
+    <Flex
+      flexDirection="column"
+      gap="5"
+      alignItems={{ base: 'center', md: 'flex-start' }}
+      paddingX="1"
+    >
       <Description
         description="Initial: Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s"
         certRequested={false}
         validFrom={validFrom}
         validTo={validTo}
       />
-      <Flex flexDirection="column" gap="1">
-        <Heading as="h3" size="md" marginBottom="3">
+      <Flex
+        flexDirection="column"
+        fontSize={{ base: 'sm', md: 'md' }}
+        gap={{ base: '3', md: '' }}
+        alignItems={{ base: 'center', md: 'flex-start' }}
+        textAlign={{ base: 'center', md: 'left' }}
+      >
+        <Heading as="h3" size={{ base: 'sm', md: 'md' }}>
           Domain Name
         </Heading>
         <Text>Lorem Ipsum is simply dummy text of the printing and typesetting industry</Text>
-
-        <HStack>
-          <Text fontWeight="bold">Domain Name:</Text> <Text>{subject}</Text>
-        </HStack>
+        <Flex
+          flexDirection={{ base: 'column', md: 'row' }}
+          alignItems={{ base: 'center', md: 'flex-start' }}
+        >
+          <Text fontWeight="bold">Domain Name:&nbsp;</Text>
```

Comment:
To me, it's a bit strange that many `Domain Name` are in different sizes. Perhaps it could be better to use a table or bullets without title? 

<img width="590" alt="Screenshot 2023-02-21 at 10 08 44 PM" src="https://user-images.githubusercontent.com/59734889/220511337-93efc9b0-32f8-4a42-9bd1-7d953a7b5396.png">

Reply:
I'm actually a bit confused about this. If we're going to display the `*` is there really a need to show all domains?

Reply:
I'm actually struggling to understand too. Probably worth discussing in a meeting.

Reply:
For the certificate there should only be one domain shown, the root domain like
`*.username.senecastudent.ca`

Reply:
I was actually talking about repeated same titles. I don't know what content will be under big title, but it seems strange to have the same title twice. I guess I'm the only one who's sensitive on this. I think it's opinionated and it shouldn't be an issue.